### PR TITLE
Adding crush removal step

### LINKED ISSUE #745: [doc] 3.3  Removing Nodes
### Type of report
   - [X] bug
   - [ ] feature request

### Source Document
3.3  Removing Nodes:

https://susedoc.github.io/doc-ses/master/html/ses-admin/storage-salt-cluster.html#salt-node-removing

### Summary

From:

https://tracker.ceph.com/issues/48260
https://bugzilla.suse.com/show_bug.cgi?id=1176265

After removing all OSDs on a single host, the user has to manually remove the "host" from the crush map.

### PR DESCRIPTION
**Pull request by asettle**
Closes #745

### CODE REVIEW

File: xml/admin_saltcluster.xml [hunk 1]
```diff
@@ -200,6 +200,17 @@ placement:
...
 <screen>&prompt.cephuser;ceph-salt config /ceph_cluster/roles/bootstrap reset</screen>
    </step>
+   <step>
+    <para>
+      After removing all OSDs on a single host, remove the host from the crush map:
+    </para>
+<screen>&prompt.cephuser;ceph osd crush remove <replaceable>bucket-name</replaceable></screen>
+    <note>
+      <para>
+        The bucket name should be the same as the host name.
```
Comment (asettle) [inline xml/admin_saltcluster.xml:210]: @jecluis if you think we need to be more verbose here, let me know. It's a little vague, but I do feel by this step they should be aware of how to remove this. Might be too much assumed knowledge, however.

File: xml/admin_saltcluster.xml [hunk 2]
```diff
@@ -200,6 +200,17 @@ placement:
     </para>
 <screen>&prompt.cephuser;ceph-salt config /ceph_cluster/roles/bootstrap reset</screen>
    </step>
+   <step>
+    <para>
+      After removing all OSDs on a single host, remove the host from the crush map:
```
Comment (lproven) [inline xml/admin_saltcluster.xml:205]:
```suggestion
      After removing all OSDs on a single host, remove the host from the CRUSH map:
```

Comment (lproven) [reply xml/admin_saltcluster.xml:205]: CRUSH is an acronym, isn't it?

**Review by lproven — CHANGES_REQUESTED** [review]

**Review by lproven — APPROVED** [review]

File: xml/admin_saltcluster.xml (hunk 1 above)
Comment (jecluis) [reply xml/admin_saltcluster.xml:210]: I think this is clear enough. Unless ones does not read the surrounding text and just goes about running commands blindly, but I think we can hardly blame the docs for that.

**Review by jecluis — APPROVED** [review]

**Pull request merged**